awx/playbooks/project_update.yml

```diff
@@ -187,22 +182,19 @@
         - install_roles
 
     - block:
-        - name: detect collections/requirements.(yml/yaml)
-          stat:
-            path: "{{project_path|quote}}/collections/requirements{{ item.ext }}"
-          with_items: "{{ yaml_exts }}"
-          register: doesCollectionRequirementsExist
-
         - name: fetch galaxy collections from collections/requirements.(yml/yaml)
           command: >
-            ansible-galaxy collection install -r {{ item.stat.path }}
+            ansible-galaxy collection install -r {{ item }}
             --collections-path {{projects_root}}/.__awx_cache/{{local_path}}/stage/requirements_collections
             {{ ' -' + 'v' * ansible_verbosity if ansible_verbosity else '' }}
           args:
             chdir: "{{project_path|quote}}"
           register: galaxy_collection_result
-          with_items: "{{ doesCollectionRequirementsExist.results }}"
-          when: item.stat.exists
+          with_fileglob:
+            - "{{project_path|quote}}/collections/requirements.yaml"
+            - "{{project_path|quote}}/collections/requirements.yml"
+            - "{{project_path|quote}}/requirements.yaml"
+            - "{{project_path|quote}}/requirements.yml"
```

Comment:
@geerlingguy with https://github.com/ansible/awx/issues/106 in mind, am I getting hotter or colder?

Did Ansible itself roll out a combined `ansible-galaxy install` command that handled both collections and roles? If it did, then using that for the top-level requirements would be preferable.

Reply:
Was thinking about https://github.com/ansible/ansible/pull/67843 by @jborean93 

and after reading back over that, I believe there's nothing we can really use from it. We still support Ansible 2.9 (which doesn't have the combined command), and we have to specify the roles path, which isn't compatible with the combined roles & collections install command.

So I think this answers the question for me - we have to keep using `ansible-galaxy` `role`/`collection` `install` forever.

Now I'm questioning if we should install roles from `{{project_path|quote}}/requirements.yaml` or not.